schedule.md

```diff
@@ -1,3 +1,13 @@
 | Unit |  Date      | Lesson |
 |:----:|:----------:|--------|
 | 0    | 2016-07-18 | [Environment & Tools](lessons/env-and-tools) |
+| 0    | 2016-07-18 | [Intro to Pair Programming]() |
+| 1    | 2016-07-19 | [JS Fundamentals: Workshop Review](lessons/javascript-fundamentals/workshop-review) |
+| 1    | 2016-07-20 | [JS Fundamentals: Types, Strings, and Numbers](lessons/javascript-fundamentals/types-strings-and-numbers) |
+| 1    | 2016-07-21 | [JS Fundamentals: Objects and Arrays](lessons/javascript-fundamentals/objects-and-arrays) |
+| 1    | 2016-07-22 | [Intro To HTML and CSS Part 1](lessons/html-and-css/intro-to-html-and-css-part-1) |
+| 1    | 2016-07-25 | [JS Fundamentals: Functions Deep Dive](lessons/javascript-fundamentals/functions-deep-dive) |
+| 1    | 2016-07-26 | [JS Fundamentals: Advanced Array Methods](lessons/javascript-fundamentals/advanced-array-methods) |
+| 1    | 2016-07-27 | [Intro To HTML and CSS Part 2](lessons/html-and-css/intro-to-html-and-css-part-2) |
+| 1    | 2016-07-28 | [Review/Project]() |
+| 1    | 2016-07-29 | [Asessment]() |
```

Comment:
Two General Thoughts/ Comments:

1. What are the overall learning standards you are working towards? I see some terrific particular learning goals, but it’s also helpful to situate those in an overarching arc. How do these specific daily goals inform a larger, longer-scale learning goals? What are we aiming for students to be able to do by the end of the unit?

2. What are students doing during "demo" time? Is this live coding by you? Are the students typing along? 

Thoughts on things to keep engagement up:
-plan challenging questions to intersperse throughout, especially evaluative and analytical questions (eg. when would we use this data structure over that one; why doesn’t X work?)
-intentionally make mistakes, have students help identify mistakes
-students research terms using google and then share w/ groups (instead of demo)
-have students prepare a live coding demonstration on the previous day’s content

Thanks for sharing all of this, Nate! Looking good.